### ISSUE TITLE
Забыл указать memo поле

### ISSUE THREAD

**Issue by varoffhellss** [toncenter/ton-wallet#44]
При переводе с TON Wallet на FTX забыл указать memo поле, что делать?

**Comment by golovin-evgenii** [comment]
У меня такая же история, я указал ошибочный мемо (
Вам удалось решить вопрос?

**Comment by Shaitan777** [comment]
К сожалению нет. Отправлено с моего телефона HONOR-------- Исходное сообщение --------От: golovin-evgenii ***@***.***>Дата: вт, 28 дек. 2021 г., 11:10Кому: toncenter/ton-wallet ***@***.***>Копия: Subscribed ***@***.***>Тема: Re: [toncenter/ton-wallet] Забыл указать memo поле (Issue #44)

У меня такая же история, я указал ошибочный мемо (

Вам удалось решить вопрос?

**Comment by designervoid** [comment]
тут только в саппорт биржи писать, я полагаю 😔

**Comment by golovin-evgenii** [comment]
Да, я написал в саппорт биржи и они мне в качестве исключения зачислили
тоны на кошелёк.

сб, 1 янв. 2022 г., 02:11 Maxim Chernyshov ***@***.***>:
